### PR TITLE
Explain caliper tracking app through commnets in aws.py

### PR DESCRIPTION
**Pull request by HamzaIbnFarooq**
**Description:** _Explain caliper tracking app through commnets in aws.py_

**Checks before merge:**

- [ ] Reviewed
- [ ] Commits squashed

### CODE REVIEW

File: lms/envs/aws.py [hunk 1]
```diff
@@ -1099,6 +1099,18 @@
 INSTALLED_APPS.append('openedx.features.ucsd_features')
 
 ############### Settings for Caliper Tracking #####################
+# 'caliper_tracking' app enables us to transform Edx event-logs according to
+# IMSGlobal Caliper Standards.
+# https://www.imsglobal.org/sites/default/files/caliper/v1p1/caliper-spec-v1p1/caliper-spec-v1p1.html.
+#
+# Dependencies:
+#   1. This app adds its processor in both the event tracking engine's settings
```
Comment (asamolion) [inline lms/envs/aws.py]: Please make this more descriptive. Explain exactly what setting it changes in `EVENT_TRACKING_BACKEND` AND `TRACKING_BACKENDS` and what is the difference between them.

File: lms/envs/aws.py [hunk 2]
```diff
@@ -1099,6 +1099,18 @@
...
+#      [EVENT_TRACKING_BACKENDS, TRACKING_BACKENDS]
+#   2. This app must be placed in INSTALLED_APPS settings before
```
Comment (asamolion) [inline lms/envs/aws.py]: This line is unnecessary because the caliper_app already does this automatically. It only needs to be placed in installed apps.

Comment (HamzaIbnFarooq) [reply lms/envs/aws.py]: I wrote this to explain why our code is placing 'caliper_tracking' above 'eventtracking' app.

File: lms/envs/aws.py [hunk 3]
```diff
@@ -1099,6 +1099,39 @@
 INSTALLED_APPS.append('openedx.features.ucsd_features')
 
 ############### Settings for Caliper Tracking #####################
+# 'caliper_tracking' app enables us to transform Edx event-logs according to
+# IMSGlobal Caliper Standards.
+# https://www.imsglobal.org/sites/default/files/caliper/v1p1/caliper-spec-v1p1/caliper-spec-v1p1.html.
+#
+# Following are the changes made on edx by caliper_tracking app.
```
Comment (asamolion) [inline lms/envs/aws.py]: `Following are the changes made in Django settings by caliper_tracking app.`

File: lms/envs/aws.py [hunk 4]
```diff
@@ -1099,6 +1099,39 @@
...
+
+# 1. EVENT_TRACKING_BACKENDS settings:
+#      It is responsible for logging events of 'eventtracking' app, our
+#      'caliper_tracking' app appends its custom processor (named
```
Comment (asamolion) [inline lms/envs/aws.py]: remove the word `named`

File: lms/envs/aws.py [hunk 5]
```diff
@@ -1099,6 +1099,39 @@
...
+#      'caliper_tracking.processor.CaliperProcessor') in the pipeline of
+#      'processors' present in 'tracking_logs' of 'EVENT_TRACKING_BACKENDS'.
```
Comment (asamolion) [inline lms/envs/aws.py]: the `processors` key is present under `OPTIONS` in `tracking_logs`

File: lms/envs/aws.py [hunk 6]
```diff
@@ -1099,6 +1099,39 @@
...
+#
+# 2. TRACKING_BACKENDS settings:
+#      Logs generated by 'track' app use 'TRACKING_BACKENDS' settings.
+#      'caliper_tracking' replaces its processor with its own custom backend
```
Comment (asamolion) [inline lms/envs/aws.py]: `replaces its default logging backend with`

File: lms/envs/aws.py [hunk 7]
```diff
@@ -1099,6 +1099,39 @@
...
+#      (named 'CALIPER_TRACKING_BACKENDS') which eventually redirects the logs
```
Comment (asamolion) [inline lms/envs/aws.py]: It is not named `CALIPER_TRACKING_BACKEND`. Please use the correct name.

File: lms/envs/aws.py [hunk 8]
```diff
@@ -1099,6 +1099,39 @@
...
+#        'logger': {
+#          'ENGINE': 'caliper_tracking.processor.CaliperProcessor',
+#          'OPTIONS': {
+#             'name': 'tracking'
+#          }
+#        }
+#      }
+#
+# 3. This app must be placed in INSTALLED_APPS settings before
+#    'eventtracking' app, as 'eventtracking' app uses EVENT_TRACKING_BACKENDS
```
Comment (asamolion) [inline lms/envs/aws.py]: `as 'eventtracking' app uses EVENT_TRACKING_BACKENDS settings which must be changed earlier by 'caliper_tracking' app.` 

This part is unnecessary.

File: lms/envs/aws.py (hunk 3 above)
Comment (HamzaIbnFarooq) [reply lms/envs/aws.py]: done

File: lms/envs/aws.py (hunk 4 above)
Comment (HamzaIbnFarooq) [reply lms/envs/aws.py]: done

File: lms/envs/aws.py (hunk 5 above)
Comment (HamzaIbnFarooq) [reply lms/envs/aws.py]: done

File: lms/envs/aws.py (hunk 6 above)
Comment (HamzaIbnFarooq) [reply lms/envs/aws.py]: Done

File: lms/envs/aws.py (hunk 7 above)
Comment (HamzaIbnFarooq) [reply lms/envs/aws.py]: done

File: lms/envs/aws.py (hunk 8 above)
Comment (HamzaIbnFarooq) [reply lms/envs/aws.py]: done

File: lms/envs/aws.py [hunk 9]
```diff
@@ -1099,6 +1099,38 @@
 INSTALLED_APPS.append('openedx.features.ucsd_features')
 
 ############### Settings for Caliper Tracking #####################
+# 'caliper_tracking' app enables us to transform Edx event-logs according to
```
Comment (arooshaarif) [inline lms/envs/aws.py]: 'allows us' would be more suitable here instead of 'enables us'

**Pull request merged**